styles.css

```diff
@@ -107,3 +107,450 @@ a {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.7rem;
+  gap: 0.6rem;
+  border: none;
+  background: #36b37f;
+  border-radius: 0.2rem;
+}
+
+.see-project-text {
+  font-weight: 500;
+  font-size: 1.6rem;
+  line-height: 1.5em;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4rem 1.5rem;
+  gap: 3.6rem;
+  background: url("Assests/Images/about.svg");
+  background-repeat: no-repeat;
+  margin: 0;
+}
+
+.about-me-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 1rem;
+  width: 351px;
+  height: 333px;
+  margin: 0;
+}
+
+.about-me-title {
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 3.2rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  margin: 0;
+}
+
+.about-me-content {
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 2rem;
+  text-align: center;
+  color: #42526e;
+  margin: 0;
+}
+
+.btn-resume {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.7rem;
+  gap: 0.6rem;
+  width: 159px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 0.2rem;
+  border: none;
+  margin-top: 1rem;
+}
+
+.get-my-resume {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.about-me-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1px;
+  gap: 24px;
+  width: 327px;
+  height: 1106px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.about-me-cards {
+  width: 327px;
+  height: 352px;
+}
+
+.about-img-placeholder {
+  box-sizing: border-box;
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+}
+
+.icons {
+  display: flex;
+  justify-content: center;
+  width: 327px;
+  height: 128px;
+  left: calc(50% - 327px / 2);
+  top: 16px;
+}
+
+.img-icon {
+  margin-top: 2.5rem;
+}
+
+.title-and-tags {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  bottom: 72px;
+  margin-top: 3rem;
+}
+
+.card-title {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  color: #344563;
+  margin: 0;
+}
+
+.title-tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 271px;
+  height: 40px;
+  margin-top: 0;
+  margin-bottom: 4.5rem;
+  flex-wrap: wrap;
+}
+
+.title-tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 10px;
+  width: 89px;
+  height: 40px;
+  background: #fff;
+  border-radius: 8px;
+  margin: 0;
+  flex: 1;
+}
+
+.title-tag-text {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #36b37e;
+}
```

Comment:
- [ ] Kindly make sure your **technologies cards** follow the design like the ones from the Figma template. Right now they overlap the **Get My Resume** button from the **About Me section**, there is no spacing between them,...
You check all details from Figma to match your design to your chosen template.

| Figma Design | Your Design |
| ------------------ | ----------------- |
| ![techology_cards](https://user-images.githubusercontent.com/83042742/185719614-30ae2623-d40b-41a2-8bd2-0eb4631f7f9c.png) | ![Screenshot from 2022-08-20 01-32-20](https://user-images.githubusercontent.com/83042742/185719627-b71d1abb-d11c-47ce-ac53-af1408b97b2c.png) |